index.Rmd

```diff
@@ -25,7 +25,7 @@ library(scales)
 library(kableExtra)
 
 loadacs5 <- load_variables(2018, "acs5", cache = TRUE)
-#api_key <- census_api_key("type_your_API_here")
+#api_key <- census_api_key("type_your_API_here") 
```

Comment:
Obviously you'll have to have your own API key in here but I think you know that. I entered my own to run your code

Reply:
Yeah, this is just a placeholder so people can run their own script. Adam suggested I do this since I of course don't want to share my API key. Glad you noticed this!